src/services/config/configServiceV2.js

```diff
@@ -0,0 +1,48 @@
+const { Configuration } = require('@hpi-schul-cloud/commons');
+
+/**
+ *
+ */
+const configServiceHooksV2 = {
+	before: {
+		all: [],
+		find: [],
+		// get: [],
+		// create: [],
+		// update: [],
+		// patch: [],
+		// remove: [],
+	},
+	after: {
+		all: [],
+		find: [],
+		// get: [],
+		// create: [],
+		// update: [],
+		// patch: [],
+		// remove: [],
+	},
+};
+
+/**
+ * This service for the env variables to sync between server and client.
+ * These env variables must be public and there must be no secret values.
+ */
+class ConfigServiceV2 {
```

Comment:
i would prefere a different name here. configuration always might have a scope where it's defined for. either in the whole service or the find i would like to separate between when the data can be accessed. This seems to be application configuration which is publicly available. why not name it PublicApplicationConfigurationService? When later other data might follow (like User/School specific settings after authentication) we can add it separately.

why? i would prefere thinking in use cases.